src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/dto/AggregationRecord.scala

```diff
@@ -0,0 +1,12 @@
+package com.microsoft.partnercatalyst.fortis.spark.sinks.cassandra.dto
+
+trait AggregationRecord {
+  val periodstartdate: Long
+  val periodenddate: Long
```

Comment:
we use it for the aggregation. I feel like it will serve its existence for the time range filters in the histogram.